### PR TITLE
Add dependabot updates for Docker and Golang

### PR DESCRIPTION
**Pull request by detiber**
## Description

Adds dependabot updates for Docker and Golang

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -2,6 +2,14 @@ version: 2
...
+    schedule:
+      interval: "daily"
+  - package-ecosystem: "gomod"
+    directory: "/"
+    schedule:
+      interval: "daily"
+  - package-ecosystem: "docker"
     directory: "/"
     schedule:
       interval: "daily"
```
Comment (invidian) [inline .github/dependabot.yml:15]: Can we fix newline?

Comment (detiber) [reply .github/dependabot.yml]: Indeed, I also added some comments for the new addiitions.

File: .github/dependabot.yml [hunk 2]
```diff
@@ -4,4 +4,16 @@ updates:
   - package-ecosystem: "github-actions"
     directory: "/"
     schedule:
-      interval: "daily"
\ No newline at end of file
+      interval: "daily"
+
+  # Maintain dependencies for Go modules
```
Comment (invidian) [inline .github/dependabot.yml:9]:
```suggestion
  # Maintain dependencies for Go modules.
```

File: .github/dependabot.yml [hunk 3]
```diff
@@ -4,4 +4,16 @@ updates:
...
+  - package-ecosystem: "gomod"
+    directory: "/"
+    schedule:
+      interval: "daily"
+
+  # Maintain dependencies for Docker images
```
Comment (invidian) [inline .github/dependabot.yml:15]: I always suggest writing all comments as sentences, then there is no arguing about capitalization and dots :smile: 
```suggestion
  # Maintain dependencies for Docker images.
```

**Review by invidian — COMMENTED** [review]
Not approving, as I have no experience with dependabot :)

**Pull request merged**